app/test_main.py

```diff
@@ -1,3 +1,25 @@
 from app.main import get_human_age
 
-# write your code here
+
+def test_should_return_correct_human_age_for_cat_and_dog() -> None:
+    assert get_human_age(15, 15) == [1, 1]
+
+    assert get_human_age(24, 24) == [2, 2]
+
+    assert get_human_age(28, 15) == [3, 1]
+
+    assert get_human_age(15, 29) == [1, 3]
+
+    assert get_human_age(36, 35) == [5, 4]
+
+    assert get_human_age(10, 10) == [0, 0]
```

Comment:
Consider using `@pytest.mark.parametrize` to streamline these test cases. This will make your tests cleaner and easier to maintain, as suggested in the checklist.